code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -256,9 +259,23 @@
 
 /mob/living/simple_animal/pet/cat/cak/attack_hand(mob/living/L)
 	..()
-	if(L.a_intent == INTENT_HARM && L.reagents && !stat)
-		L.reagents.add_reagent("nutriment", 0.4)
-		L.reagents.add_reagent("vitamin", 0.4)
+	if(L.a_intent != INTENT_HARM && L.a_intent != INTENT_DISARM)
+		return
+	if(!ishuman(L))
+		return
+	var/mob/living/carbon/human/H = L
+
+	if(stat == DEAD)
+		final_bites++
+		if(final_bites >= total_final_bites)
+			visible_message("<span class='danger'>[H] finished eating [src], there's nothing left!</src>")
+			if(H.dna && H.dna.species.reagent_tag == PROCESS_ORG)
+				to_chat(L, "<span class='info'>That last bite didn't taste like cake.  You have a bad feeling about this....")
+				L.reagents.add_reagent("prions", 5)
```

Comment:
This is going to exclusively get inexperienced players and noone else

Reply:
Swapped for Teslium per discussion.